src/fetching/getJourneys.ts

```diff
@@ -0,0 +1,10 @@
+import { defaultFetch } from '.';
+import { ZetkinJourney } from '../types/zetkin';
+
+export default function getJourneys(orgId: string, fetch = defaultFetch) {
+  return async (): Promise<ZetkinJourney[]> => {
+    const journeysRes = await fetch(`/orgs/${orgId}/journeys`);
+    const journeysBody = await journeysRes.json();
+    return journeysBody?.data;
+  };
+}
```

Comment:
Comrade, we aren't using these fetching functions anymore, we are using our fancy new resource handlers. Check out `src/api` to see how. Happy to help if it's confusing!

Reply:
Just made an update!

Reply:
Hey @ziggabyte just jumping in here - the journeys resource looks good, so maybe you can delete the file here in /fetching? :-D

Reply:
@lowlandjuju excellent suggestion! 😄